trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/results/ReqCheckRedundancyResult.java

```diff
@@ -0,0 +1,22 @@
+package de.uni_freiburg.informatik.ultimate.pea2boogie.results;
+
+import de.uni_freiburg.informatik.ultimate.core.model.models.IElement;
+
+public class ReqCheckRedundancyResult<LOC extends IElement> extends ReqCheckFailResult<LOC> {
+
+	private final String mReqName;
+	private final String mRedundancySet;
```

Comment:
Are you really sure you want to store a set as a string here?

Reply:
Changed to a set and using the toString() method inside the description method, that should be better I think.

https://github.com/ultimate-pa/ultimate/blob/d0ca74a59630253c0b49eada9bd9e9732a9ce6dc/trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/results/ReqCheckRedundancyResult.java#L37

https://github.com/ultimate-pa/ultimate/blob/d0ca74a59630253c0b49eada9bd9e9732a9ce6dc/trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/results/ReqCheckRedundancyResult.java#L48